src/config/entities/configuration.ts

```diff
@@ -17,6 +17,9 @@ export default () => ({
   applicationPort: process.env.APPLICATION_PORT || '3000',
   auth: {
     token: process.env.AUTH_TOKEN,
+    nonceTtlSeconds: parseInt(
+      process.env.AUTH_NONCE_TTL_SECONDS ?? `${60 * 60}`,
+    ),
```

Comment:
Is there a reference or recommended minimum value from the spec?

Reply:
No, this is an arbitrary value. It is effectively the period when a user should sign the message. Do you think we should decrease it?

I considered setting it to the `expirationTime` but that value is not always present and the nonce is cleared if the message is expired.

Reply:
As per discussion, this was reduced to five minutes in 8c6934c772570d3a66c079a042e3cf28af7a1f6e.